docs/schema/codelists.md

```diff
@@ -121,7 +121,11 @@ The submission method codelist is used to identify the mechanism through which a
 ```{versionadded} 1.1
 ```
 
-The related process block is used at the release level to point backwards to prior processes, such as planning or framework establishment, and at the contract level to point onwards to subcontracts or to renewal or replacement processes. The related process codelist determines the kind of relationship that is being described.
+```{versionchanged} 1.2
+Deprecated "parent-to-child" codes: 'subContract', 'replacementProcess' and 'renewalProcess'. Added sub-codes of prior: 'prior.replacement', 'prior.renewal', 'prior.additional' and 'prior.repetition'.
+```
+
+The related process block is used at the release level to point backwards to prior processes, such as a planning process or the first stage of a framework agreement procedure. The related process codelist determines the kind of relationship that is being described. For more information, see [contracting processes and planning processes](../guidance/map/contracting_planning_processes.md) and [framework agreements](../guidance/map/framework_agreements.md).
```

Comment:
````suggestion
```{field-description} ../../../build/current_lang/release-schema.json /definitions/RelatedProcess
```

```{seealso}
* [Map: Contracting processes and planning processes](../guidance/map/contracting_planning_processes.md)
* [Map: Framework agreements](../guidance/map/framework_agreements.md)
```
````